### ISSUE TITLE
Working group web page

### ISSUE THREAD

**Issue by jclingan** [microprofile/microprofile-marketing#266]
This issue relates to eclipse/microprofile#178. If [MicroProfile.io](https://microprofile.io/) is used to house working group information, then it will need a working group tab with links to resources, charter, working group members, etc.

**Comment by aeiras** [comment]
As stated on the last MP Community hangout last week April 28th, I will be attending May MP Marketing hangouts on Monday 10am PDT to attend with the group the WG 3 tasks that need to be worked in parallel with the WG Charter. :)

**Comment by aeiras** [comment]
New Working Group tab: _Working Group_

**minimum listed items:** by name: 

- Charter -  link

- Participation Agreement -  link

- Individual Agreement -  link

- Employer Consent Agreement - link

- Trademark Guidelines - MP wiki link

- Q&A - MP wiki link

- Members Onboarding  - MP wiki link

- Collateral contains:

1. marketing documents, currently 2 associated with MPWG
2. logos

**Comment by aeiras** [comment]
@rstjames :) I will add ticket to today's call but I don't think there will be time to discuss it there. This ticket is enough to get us started with the creation of the Tab. I think!

**Comment by rstjames** [comment]
@aeiras Not sure I understand what is needed here. Think a meeting is needed to move this forward.

**Comment by aeiras** [comment]
No rush, next week during the MPWG we can discuss this ticket Ryan. 

A new tab under the current website is needed to host all data about the Working Group. It is the source of truth of the WG per say.

**Comment by Emily-Jiang** [comment]
We need to add the following:

-  Meeting minutes (e.g. [Jakarta EE Meeting Minutes](https://jakarta.ee/meeting_minutes/))

- Become a member (e.g. https://jakarta.ee/membership/)

- Stay Connected (e.g. https://jakarta.ee/connect/)

**Comment by aeiras** [comment]
after today's mktg video call, list improves in addition to Emily's items above:

MPWG Charter 

[MPWG Individual Participation Agreement ](https://www.eclipse.org/org/workinggroups/wpga/microprofile-working-group-participation-agreement-individual.pdf)

Individual Agreement - link

Employer Consent Agreement - link

Trademark Guidelines - MP wiki link

Q&A - MP wiki link

Members Onboarding - MP wiki link

Collateral contains:

marketing documents, currently 2 associated with MPWG
logos

**Comment by aeiras** [comment]
Ryan, 

Can you please double check that all links has public access? Thanks!
Ordering, let's work on that a bit later. 

After today's mktg video call, list improves to subgroups and more items: 

**General:** 

- [MPWG Charter](https://www.eclipse.org/org/workinggroups/microprofile-charter.php)
- MPWG Specification Process - **WIP**

- logos listed by Alphabetic order including the 3 Memberships: Corporate, Committers (quotes feed rotate- how can we do that?), Guests


- Members', Committers' and Guests' Quotes -- rotate every 30seconds or so. 
**Legal**
_The Agreements are executed between the Party and the Eclipse Foundation__

- [Eclipse Foundation Contributor Agreement, ECA](https://www.eclipse.org/legal/ECA.php). 

- [MPWG Individual Participation Agreement](https://www.eclipse.org/org/workinggroups/wpga/microprofile-working-group-participation-agreement-individual.pdf)

- [MPWG Participation Agreement Agreement ](https://www.eclipse.org/org/workinggroups/wpga/microprofile-working-group-participation-agreement.pdf)

- [Employer Consent Agreement for Eclipse Foundation Specification Projects](https://www.eclipse.org/legal/EmployerConsentAgreementForSpecificationProjects-v1.0.pdf)

- [Eclipse Foundation Trademark Usage Guidelines ](https://www.eclipse.org/legal/logo_guidelines.php)

- MPWG Trademarks -  **WIP** **later** 

**Budget**

- MPWG Budget  **WIP** **later** 

- MPWG Marketing Budget   **WIP** **later** 

**Knowledge**

- MPWG Questions & Answers -   ´**leave it blank** I need to add it to the MP main repo to welcome PR. any link  in the google drive ought not to be listed here until properly imported. 

**Maintenance**
- Members Onboarding Support- **WIP**  **leave it blank** I need to transfer it to MP repo

- Steering Public Call Minutes, categorized by Day/Month/Day -- **WIP** MPWG will have 4 documents. 

- Steering Public Call Zoom access- **WIP**

- Steering Committee members listed and link to either individual's bio or twitter (we will use issues to update this part) **WIP**

**Subscribe**
- [MicroProfile Community Google Forum ](https://groups.google.com/forum/#!forum/microprofile)

- [MicroProfile Marketing Google Forum](https://groups.google.com/forum/#!forum/microprofile-marketing)

- MPWG Mailing list -**WIP** link later

- MPWG Steering Mailing list -**WIP**  link later

**Branding**

- MPWG Prospectus- **WIP**  link later

- MPWG Membership Flyer -**WIP**  link later

- MPWG logos "Members, Compatible, etc

**Comment by aeiras** [comment]
@rstjames, 

The quote document has been created and is inside the MPWG folder :) 
I have transferred the Prospectus quotes to the document. The pattern is 2 to 3 sentences max. 
The quote section minimal requirements:

-  rotate (at least 30seconds time)  

- to be added 1st to the MPWG new page 

https://docs.google.com/spreadsheets/d/14iCNpzj7BpvntOKG3u2ooEa-7LFceZTqk9OmqVqbDog/edit#gid=0

**Comment by aeiras** [comment]
On behalf of Ryan, **below is a progress update:**

https://microprofile.io/workinggroup/
password: mpworkinggroup
I think we need to dive deeper into the content. (A lot of links)
the page has a style layout for the content we have now.

**Comment by rstjames** [comment]
Document to work on content for this page. Please put what you feel you should be there and we can chip away at it until we have a solid first version. 

Right now I feel the issue is that all we have are links. And not all links have content yet. 

[https://docs.google.com/document/d/1Th7O12AspPUWvudPVSTIZnUXXtyGVxDw-YOzUQyQMIY/edit?usp=sharing](url)

**Comment by aeiras** [comment]
@rstjames, 

The link brings us to the git repo and not the drive. Can you add the proper link? It is going to be great to help fill out the page :)

**Comment by rstjames** [comment]
@jclingan @aeiras @Emily-Jiang 

I updated the website from the last meeting. 

- added content from slides

- added CTA to the top card. ( linked buttons yet) 

- Added the MP calendar to the "Meeting Calendar" tab. It works pretty well. 

Best first we decided what we want visitors to accomplish on this page and after organizing the content more so it has a good flow. Then guide them from each section to additional information (tabs).

**Comment by aeiras** [comment]
@rstjames, it looks wonderful & it is live! :) 

**Priority 1:** 
Please create the MPWG tab as a standalone. The page must not be hidden under other tabs, it must stand on its own.  and be easily reachable. 
**Priority 2**
lets send a note to the MP forum once the page is ready . :) I think you must get the honors! 

Also, can you help update the MP quotes? A few more have arrived:  https://groups.google.com/forum/#!topic/microprofile-marketing/YH6JtFHtd5A & the prospects quotes also require attention. 

So fun!

**Comment by rstjames** [comment]
@aeiras the page is not ready... there is a lot of work still needed on the content.  After this content is polished we can add it to the menu.

**Comment by rstjames** [comment]
@aeiras Content doc:
https://docs.google.com/document/d/1Th7O12AspPUWvudPVSTIZnUXXtyGVxDw-YOzUQyQMIY/edit?usp=sharing

**Comment by Emily-Jiang** [comment]
> @jclingan @aeiras @Emily-Jiang
> 
> I updated the website from the last meeting.
> 
>     * added content from slides
> 
>     * added CTA to the top card. ( linked buttons yet)
> 
>     * Added the MP calendar to the "Meeting Calendar" tab. It works pretty well.
> 
> 
> Best first we decided what we want visitors to accomplish on this page and after organizing the content more so it has a good flow. Then guide them from each section to additional information (tabs).

Thank you @rstjames ! I think the content is good. My immediate feeling is that it is too busy, too much words. It is best to use some pictures and move some text to the links somehow.

**Comment by aeiras** [comment]
+1 @Emily-Jiang on too many words. 
Thankfully @rstjames has shared the document that enables review and to cut much content. Thank you, Ryan, for sharing the link. Can you please drop it in the MPWG folder "Website'" - Let's create a new folder for website stuff under the main MPWG file. :) 

I will dedicate time this week to help with the review. Though we have reviewed much of the content, it is time to simplify it. : )

When I say "public," the page is public since launched as anyone can use the password to check it out. Sorry for the confusion. I meant that I would like for it to be already placed in its own tab and not hidden. There is no need for it to not be already public. as much as we work on it this week, it will be 3 months since we started, it must become public to enable others' eyes to arrive and help with feedback. :)

**Comment by aeiras** [comment]
[MPWG Website page](https://drive.google.com/drive/u/0/folders/16_bZXJ1zHQDnLBHyg4kAn4T3PJL1vFP5)

**Comment by aeiras** [comment]
@rstjames once you drop the document there, I will gain access to the document. I won't request access and it bothers me to have to know which email to use. :) Thanks Triber!

**Comment by rstjames** [comment]
The webpage content has been moved. 
https://docs.google.com/document/d/1Th7O12AspPUWvudPVSTIZnUXXtyGVxDw-YOzUQyQMIY/edit?usp=sharing

**Comment by aeiras** [comment]
Ryan, 

Thank you for providing the document. I started the feedback using:suggested mode:  :) 

Page links missing on:

**Call to Action** buttons:

- [Become a Member ](https://github.com/eclipse/microprofile/blob/master/CONTRIBUTING.adoc)

- [Join the Discussion](https://groups.google.com/g/microprofile)

**Comment by aeiras** [comment]
Can you please update the quotes to what it was discussed in the marketing forum? and also add the new quotes awaiting import?  Thank you for tackling all aspects of this page.

**Comment by rstjames** [comment]
@aeiras I do not understand your last comments. I do not remember agreeing to change the quote section on this page. I remember discussions about adding other committer quotes but the location of the quotes was not agreed upon or discussed in detail.

**Comment by aeiras** [comment]
Hey Ryan, 

The MPWG Prospectus document is what a considering ORG brings back to their teams to make the case for them to join MPWG. 

The quotes as they stand need to include everyone without labels or separation from Contributors to committers to guests to ORGs. 

What we don't want to see is some think they are better than others. MP protects its flat structure and does so via actions. This includes how quotes show up. 

Thanks for allowing this follow up,

**Comment by aeiras** [comment]
Hey Ryan, 

The MPWG Prospectus document is what a considering ORG brings back to their teams to make the case for them to join MPWG. 

The quotes as they stand need to include everyone without labels or separation from Contributors to committers to guests to ORGs. 

What we don't want to see is some think they are better than others. MP protects its flat structure and does so via actions. This includes how quotes show up. 

Thanks for allowing this follow up,

**Comment by aeiras** [comment]
@rstjames, 

The information discussed on today's call is now a part of the MPWG page document & ready for your.

**Comment by rstjames** [comment]
@aeiras @jclingan @Emily-Jiang 
I update the webpage based on our last call. Please let me know what you think and what changes are needed. 
https://microprofile.io/workinggroup/
password: mpworkinggroup

**Comment by aeiras** [comment]
Thank you Ryan for pulling is back in with the adjustments from yesterday's discussion. I will use comments as I consume changes to provide feedback. 

** Call to action buttoms**:

- Become a Member needs to link to the Prospectus document not Jakarta EE.

**Comment by aeiras** [comment]
Quotes:

- Prospectus quotes-- Author pictures need to be changed to company logos. 

- new quotes as listed in the drive excel sheet need to be added. The sheet holds the authors' pic via git issue. Everything is in 1 document. 
As stated yesterday, we will use PR to submit MicroProfilers quote using our work on the Jakarta EE 9 Contributors card code. :) But until them, your time and effort is needed Rtyan. Thanks!

**Comment by aeiras** [comment]
Aside from the 2 adjustments needed above, the layout and all the links look great @rstjames. Thank you for the prompt turnaround. If you can get the fixes by Friday, I am +1 to page going live on Friday, Dec 11th. Let's do this!

**Comment by aeiras** [comment]
Aside from the 2 adjustments needed above, the layout and all the links look great @rstjames. Thank you for the prompt turnaround. If you can get the fixes by Friday, I am +1 to page going live on Friday, Dec 11th. Let's do this!

**Comment by Emily-Jiang** [comment]
First of all, this looks a lot nicer. Great work @rstjames ! I have a couple of comments:

- Why should you consider joining MicroProfile? I think it is much clearer if we change to 

Why should you consider joining MicroProfile Working Group?

- Releases tab => Release Guildline
as Releases indicate they are about MicroProfile releases instead of the process etc.

**Comment by rstjames** [comment]
recap to do:

- become a Member needs to link to the Prospectus document not Jakarta EE.

- new quotes as listed in the drive excel sheet need to be added. + logos instead of headshots

- adjust title: Why should you consider joining MicroProfile Working Group?

- Releases tab => Release Guildline

If anything else please let me know. I will make these adjustments tomorrow (Thursday).

**Comment by aeiras** [comment]
Thank you for recapping @rsjames! 
I am happy to check the page once again if you need my eyes, otherwise, you have OK to publish it after completing those items.

**Comment by aeiras** [comment]
@rstjames, 

Quotes sheet https://docs.google.com/spreadsheets/d/14iCNpzj7BpvntOKG3u2ooEa-7LFceZTqk9OmqVqbDog/edit#gid=1946347003   I will close the tickets associated with each quote as soon as the MPWG page is live. :) 

If we can be named the new tab MPWG, that would be great! Let's see if space fits for the full Working Group; however, we are disciplined into using MPWG for most topics happening in the forums. The naming convention applies here as well. 

Many thanks for pushing live the new page. Once you do, if you could send them out to the general forum for notification to everyone, that would be amazing! 

Also, closing this ticket and use smaller tickets for future work to tackle improvements and things that need to be added as work in Progress seems efficient and easier to consume.

**Comment by rstjames** [comment]
@aeiras  All updated
